rclcpp/src/rclcpp/node_interfaces/node_parameters.cpp

```diff
@@ -109,8 +109,13 @@ NodeParameters::NodeParameters(
       rclcpp::ParameterMap initial_map = rclcpp::parameter_map_from(params);
 
       // Enforce wildcard matching precedence
-      // TODO(cottsay) implement further wildcard matching
```

Comment:
If removing this line, maybe we need to add more implementations.
such as  (I am not asking you to implement it, I just thought we might keep this line)

```
asia_*:
    *:
        ros__parameters:
            applies_to_all_nodes_in_particular_namespace_with_asia: true
```

Reply:
Good point, thank you, reinstated

Reply:
FWIW, the [current design doc](http://design.ros2.org/articles/static_remapping.html#match-part-of-a-rule) explicitly disallows partial matching of tokens (e.g. `*bar`), so I don't think we should worry about supporting this case (unless the design changes).